openfga/v1/errors_ignore.proto

```diff
@@ -57,6 +57,20 @@ enum ErrorCode {
   duplicate_contextual_tuple = 2055;
 }
 
+enum AuthNErrorCode { 
+
+  no_authn_error = 0;
```

Comment:
Can we reformat these lines? Spacing seems off.

Reply:
Do you mean the newline after the first line? I am just following what the others look like. Or did you mean the tabs vs. spaces problem which I did fix. Thank you!

Reply:
Meant the spacing